content/zh/docs/concepts/storage/storage-classes.md

```diff
@@ -668,10 +668,11 @@ OpenStack 的内部驱动程序已经被弃用。请使用 [OpenStack 的外部
       diskformat: zeroedthick
     ```
 
+    `diskformat`: `thin`, `zeroedthick` 和 `eagerzeroedthick`。默认值: `"thin"`。
+
 <!--
     `diskformat`: `thin`, `zeroedthick` and `eagerzeroedthick`. Default: `"thin"`.
- -->
-    `diskformat`: `thin`, `zeroedthick` 和 `eagerzeroedthick`。默认值: `"thin"`。
+-->
```

Comment:
The convention is to keep English text as comment BEFORE Chinese translation. Please fix.
For the indentation problem, you can simply apply indentation to the comment lines, e.g. line 673 and 675.

Reply:
@tengqm Thanks for your suggestions, it's useful!